Sources/Session.swift

```diff
@@ -12,7 +12,17 @@ public struct WCURL: Hashable {
     public var bridgeURL: URL
     public var key: String
 
-    public init(topic: String, version: String, bridgeURL: URL, key: String) {
+    public static let defaultBridgeURL = URL(string: "https://safe-walletconnect.gnosis.io")!
```

Comment:
I would make it a `gnosisSafeBridgeURL` and keep the default as `bridge.walletconnect.org`

Reply:
actually, I would leave the Gnosis's bridge out.

Reply:
ok